src/index.spec.js

```diff
@@ -1,27 +1,222 @@
+import mongodb from 'mongodb';
```

Comment:
Maybe .env file for tests.. Choose which package to use using the env.
For mocking check this https://github.com/williamkapke/mongo-mock